modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -246,8 +246,11 @@ private List<IRasSearchCriteria> getCriteria(RasQueryParameters queryParams) thr
 			critList.add(toCriteria);
 		}
 		if (requestor != null && !requestor.isEmpty()) {
-			RasSearchCriteriaRequestor requestorCriteria = new RasSearchCriteriaRequestor(requestor);
-			critList.add(requestorCriteria);
+			String matchedRequestor = findMatchingRequestor(requestor);
+			if(matchedRequestor != null && matchedRequestor.isBlank()){
```

Comment:
This if-statement will only be entered if a blank requestor is matched (`&& matchedRequestor.isBlank()`)

This should've been caught in a unit test, so maybe we're missing one that goes through this code?